recommenders/popularity_recommender.py

```diff
@@ -1,26 +1,28 @@
-from recommenders import data_movies, data_ratings
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
-
-    movie_ratings = pd.merge(data_movies, data_ratings, on = 'movieId')
-    movie_ratings = movie_ratings.groupby('title')['rating'].mean().sort_values(ascending = False).head(20)
-    movie_ratings = movie_ratings.to_dict()
-    #######################################################################################################
-    response = []
-
-    for movie in movie_ratings:
-        movie_record = data_movies[data_movies.title == movie].iloc[0]
-
-        response.append({
```

Comment:
The indentation of line 18 is incorrect. It should be indented one level deeper to be inside the for loop on line 17.